app/editor/src/features/admin/ingests/configurations/FrontPage.tsx

```diff
@@ -32,6 +32,7 @@ export const FrontPage: React.FC = (props) => {
         placeholder="^sv-GLB-[A-Za-z]{3}-(?<day>[0-9]{2})(?<month>[0-9]{2})(?<year>[0-9]{4})"
         value={values.configuration.publishedOnExpression}
       />
+      <FormikCheckbox label="Publish" name="configuration.publish" />
```

Comment:
Did you make sure their are no console errors related to `configuration.publish` starting out as `undefined`?

Reply:
It seems to be fine based on the test here, thanks